src/modules/windowwalker/app/Window Walker/Components/Window.cs

```diff
@@ -202,7 +202,7 @@ public bool IsWindow
         }
 
         /// <summary>
-        /// Get a value indicating whether is the window GWL_EX_STYLE is a toolwindow
+        /// Gets a value indicating whether get a value indicating whether is the window GWL_EX_STYLE is a toolwindow

```

Comment:
In this one and in following changes: `Gets a value indicating whether get a value indicating whether`